modules/bitgo/test/v2/unit/unspents.ts

```diff
@@ -0,0 +1,148 @@
+import * as nock from 'nock';
+import * as sinon from 'sinon';
+import { common, Wallet } from '@bitgo/sdk-core';
+import { TestBitGo } from '@bitgo/sdk-test';
+import { BitGo } from '../../../src';
+
+describe('Verify string type is used for value of unspent', function () {
+  const bitgo = TestBitGo.decorate(BitGo, { env: 'test' });
+  bitgo.initializeTestVars();
+  const basecoin: any = bitgo.coin('tdoge');
+  const walletData = {
+    id: '5b34252f1bf349930e34020a00000000',
+    coin: 'tdoge',
+    keys: [
+      '5b3424f91bf349930e34017500000000',
+      '5b3424f91bf349930e34017600000000',
+      '5b3424f91bf349930e34017700000000',
+    ],
+    coinSpecific: {},
+  };
+  const wallet = new Wallet(bitgo, basecoin, walletData);
+  const bgUrl = common.Environments[bitgo.getEnv()].uri;
+  const minValueStringMatcher = sinon.match('123');
+  const maxValueStringMatcher = sinon.match.string.and(sinon.match('10000000000000123'));
+
+  describe('unspents APIs with string type minValue and maxValue', function () {
+    after(function () {
+      nock.cleanAll();
+    });
+
+    ['consolidate', 'fanout'].forEach( (manageUnspentType) => {
+      it(manageUnspentType + ' should handle string type minValue and maxValue', async function () {
+        const params = { minValue: '123', maxValue: '10000000000000123' };
+
+        const consolidateUnspentsScope = nock(bgUrl)
+          .post(`/api/v2/${wallet.coin()}/wallet/${wallet.id()}/${manageUnspentType}Unspents`, body => {
+            return minValueStringMatcher.test(body.minValue) && maxValueStringMatcher.test(body.maxValue);
+          })
+          .reply(200, {
+            txInfo: {
+              unspents: [{
+                id: 123,
+                address: 'sfajlkjad',
+                value: 1242123,
+                valueString: '1242123',
+              }],
+            },
+          });
+
+        wallet.keyIds().forEach((keyId) => nock(bgUrl)
+          .get(`/api/v2/${wallet.coin()}/key/${keyId}`)
+          .reply(200, {}));
+
+        const signTransactionStub = sinon.stub(wallet, 'signTransaction').resolves({});
+
+        const sendScope = nock(bgUrl)
+          .post(`/api/v2/${wallet.coin()}/wallet/${wallet.id()}/tx/send`, { type: manageUnspentType })
+          .reply(200, {});
+
+        await wallet[manageUnspentType + 'Unspents'](params);
+
+        const txInfo = signTransactionStub.getCall(0)?.args?.[0]?.txPrebuild?.txInfo as any;
+        txInfo?.unspents?.[0].value.should.be.a.type('bigint');
+
+        consolidateUnspentsScope.isDone();
+        sendScope.isDone();
```

Comment:
```suggestion
        consolidateUnspentsScope.done();
        sendScope.done();
```

same with other instances